### ISSUE TITLE
The underlying stack and libraries seems outdated and broken

### ISSUE THREAD

**Issue by ghost** [vseloved/wiki-lang-detect#3]
The underlying stack and libraries seems outdated and broken
also I was curious to know Is there a support for Japanese Language 

The build essential is giving error, seems like the src is outdated and broken
`W: http://220.152.42.162:80/data/05e9f1e394c802a0/archive.ubuntu.com/ubuntu/pool/main/m/manpages/manpages_4.15-1_all.deb: Automatically disabled Acquire::http::Pipeline-Depth due to incorrect response from server/proxy. (man 5 apt.conf)
E: Failed to fetch http://220.152.42.162:80/data/05e9f1e394c802a0/archive.ubuntu.com/ubuntu/pool/main/m/manpages/manpages_4.15-1_all.deb  File has unexpected size (1233600 != 83832). Mirror sync in progress? [IP: 220.152.42.162 80]
   Hashes of expected file:
Fetched 49.0 MB in 39s (1269 kB/s)
    - SHA256:0c142fd5c44cae76e2e1ad5c62d3dc8f36f1a80eaaadf9a5e60325fcfffe16ed
    - SHA1:dd3996229d03fb17f3da4002d5df030a2039fef9 [weak]
    - MD5Sum:5861e417cd275a039bbd0583c5bdbcc8 [weak]
    - Filesize:83832 [weak]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?`

**Comment by vseloved** [comment]
Yes, the library works for Japanese. In fact, it works for all 150 or so languages that are present in Wikipedia.

As for the issue you see, I'm not, actually, sure what's the problem. `manpages` debian package missing? Maybe, `sudo apt update --fix-missing` will solve the problem. In any case, doesn't seem Lisp-related...

**Comment by ghost** [comment]
Didn't solved the problem even I tried to use the diffrent ubuntu image
